api/src/main/resources/vwb-exfil-manager.yaml

```diff
@@ -0,0 +1,128 @@
+openapi: 3.0.3
+info:
+  title: Terra Exfil Manager
```

Comment:
nit, would it be better to name this `Verily Terra`? I remember during the demo Tarek gave, ppl were confused about terra being the Broad terra

Reply:
ah right, this should be Verily specific. The entire file is copied from Verily internal repo. Verily is still using Terra as project name. Changed it here.